package.json

```diff
@@ -28,6 +28,9 @@
     "tap": "^12.1.1"
   },
   "dependencies": {
+    "@types/http-errors": "^1.6.1",
+    "@types/proxy-addr": "^2.0.0",
+    "@types/vary": "^1.1.0",
```

Comment:
@mcollina , just to clarify, the requested changes were regarding using `@types` as `dependencies`, instead of `devDependencies`, or importing those types definitions inside the project itself?

Reply:
I'm exactly -1 to this approach. We should not do this.

Reply:
cc @Ethan-Arrowood

Reply:
@hspedro we'd like to keep types as simple as possible with as minimal dependencies as possible too. The solution here is to implement any relevant parts of `@types/http-errors` in the type files so we do not need to rely on the package as a dependency. If things change in the future we can update the types in this repo and ship without having to deal with any external things

Reply:
got it @Ethan-Arrowood. Thanks, will update with requested changes as soon as possible.

Reply:
@Ethan-Arrowood , I've managed to refactor the dependencies for the vast majority. But, how about the more complex types such as HttpServer that comes from `@types/nodes`, can I leave `@types/node` as a project dep or should I reimplement the `http.Server`, `http.IncomingMessage`, `http.ServerResponse`?

Reply:
Leave `@types/node` as a dependency. If users are using Fastify with TypeScript they will have it installed so it is okay for us to include.